Doc/whatsnew/3.9.rst

```diff
@@ -314,6 +314,15 @@ Added a new function :func:`gc.is_finalized` to check if an object has been
 finalized by the garbage collector. (Contributed by Pablo Galindo in
 :issue:`39322`.)
 
+hashlib
+-------
+
+Builtin hash modules can now be disabled with
+``./configure --without-builtin-hashes`` or selectively enabled with e.g.
```

Comment:
--without-builtin-hashlib-hashes perhaps just to avoid any confusion from the argument name alone that this only applies to haslib, and not the hash() builtin or the dict/set hash algorithm.

Reply:
Sounds like you have a German in your family tree! The even-longer-name-than-a-German-would-pick it is!